settings.py

```diff
@@ -13,7 +13,7 @@
 
 DEBUG = True
 TEMPLATE_DEBUG = DEBUG
-DEBUG_PROPAGATE_EXCEPTIONS = False
+DEBUG_PROPAGATE_EXCEPTIONS = True
```

Comment:
Well, there was a reason: a121c9290d28fcf27971e97344332c02ef83643a - this disables django's rendered 500 view completely afair.